Makefile

```diff
@@ -128,11 +144,21 @@ mixedindentlint: node_modules/mixedindentlint
 $(CLIENT_CONFIG_FILE): .env config/$(CALYPSO_ENV).json config/client.json server/config/regenerate-client.js
 	@$(NODE) server/config/regenerate-client.js > $@
 
-public/style.css: node_modules $(SASS_FILES)
+build/$(CLIENT_STYLE_FILES_HASH):
+	@rm -f build/*.hash
+	@touch build/$(CLIENT_STYLE_FILES_HASH)
+
+build/_components.scss: build/$(CLIENT_STYLE_FILES_HASH)
+	@mkdir -p build
+	@echo "$(CLIENT_STYLE_FILES)" | tr " " "\n" | sed "s/^client\//@import '/; s/$$/';/ ; s/\.scss//" > $@
```

Comment:
At least on my Mac, this will write the stylesheets into `_components.scss` in alphabetical order, which I expect is close to what we had previously. Do you think other environments might not necessarily write them in this order? Ideally we're not dependent upon the ordering, but... we might be 😄